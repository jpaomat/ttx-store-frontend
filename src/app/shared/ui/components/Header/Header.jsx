import { Navbar } from '../Navbar/Navbar';
import './Header.scss';

export const Header = () => {
	return (
		<header className='header-container'>
			<Navbar />
		</header>
	);
};
