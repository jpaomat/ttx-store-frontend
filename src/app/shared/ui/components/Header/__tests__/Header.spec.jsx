import { Header } from '../Header';
const Renderer = require('react-test-renderer/shallow');

const renderer = new Renderer();

describe('Techtronix component', () => {
	it('should render component', () => {
		const component = renderer.render(<Header />);
		expect(component).toMatchSnapshot();
	});
});