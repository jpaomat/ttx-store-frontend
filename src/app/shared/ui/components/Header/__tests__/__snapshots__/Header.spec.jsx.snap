// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Techtronix component should render component 1`] = `
<header
  className="header-container"
>
  <Navbar />
</header>
`;
